pkg/releaseutil/manifest.go

```diff
@@ -42,8 +42,13 @@ func SplitManifests(bigfile string) map[string]string {
 	res := map[string]string{}
 	// Making sure that any extra whitespace in YAML stream doesn't interfere in splitting documents correctly.
 	docs := sep.Split(bigfile, -1)
-	for i, d := range docs {
-		res[fmt.Sprintf(tpl, i)] = d
+	var count int = 0
+	for _, d := range docs {
```

Comment:
It used to be there and was taken out. I think we should have it and I'm happy to put it back in.